src/app/services/shared.service.ts

```diff
@@ -13,7 +13,6 @@ export class SharedService {
   public assets: BehaviorSubject<any> = new BehaviorSubject<any>(false);
   public connectionInfo: BehaviorSubject<any> = new BehaviorSubject<any>(false);
   public loginScreenSubject: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
-  public allServicesInfo: BehaviorSubject<any> = new BehaviorSubject<any>(false);
```

Comment:
is this safe to remove? Are we not polling it regularly?

Reply:
We were not using value of `allServicesInfo` anywhere, so I removed it.